dataactcore/scripts/loadHistoricalFabs.py

```diff
@@ -324,6 +456,10 @@ def main():
     logger.info("Historical FABS script complete")
 
 
+def is_nan(num):
```

Comment:
What is this doing? Pandas has an `isnull()` function, not sure if that works for single values but you could take a look?

Reply:
pd.isnull(val) works